#! /bin/sh
patch -p1 -f $* < $0
exit $?

Fix windres tool invocation.  This is the wrong patch!  The right way
to do it is to add a configure tool check for WINDRES.  However, this
requires rerunning autoconf, and this small patch is all we need to
make it build.

--- glib-2.6.6-orig/build/win32/compile-resource	2005-12-31 13:41:37.000000000 +0100
+++ glib-2.6.6/build/win32/compile-resource	2005-12-31 13:42:01.000000000 +0100
@@ -38,7 +38,7 @@
     fi
 
     m4 -DBUILDNUMBER=$buildnumber <$rcfile >$$.rc &&
-	${WINDRES-windres} $$.rc $resfile &&
+        i586-mingw32msvc-windres $$.rc $resfile &&
 	rm $$.rc
 else
     # Return failure
